pkgs/top-level/all-packages.nix

```diff
@@ -34513,4 +34513,6 @@ with pkgs;
   honeytail = callPackage ../servers/tracing/honeycomb/honeytail { };
 
   honeyvent = callPackage ../servers/tracing/honeycomb/honeyvent { };
+
+  skeema = callPackage ../tools/system/skeema {};
```

Comment:
Nitpicking: we try to roughly keep this file in alphabetical order (see the comment in the first line), can you move the line, maybe below `### TOOLS`?